src/main/java/core/basesyntax/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.TreeMap;
+
+public class Storage {
+    static HashMap<String, TreeMap<LocalDate, Integer>> fruitsInStore = new HashMap<>();
```

Comment:
let's make this field private, so we would use encapsulation here

Reply:
done